Composer/packages/ui-plugins/virtual-assistant-creation/src/components/configSummaryPage.tsx

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable format-message/literal-pattern */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import formatMessage from 'format-message';
+import React, { Fragment, useContext } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { Separator, ISeparatorStyles } from 'office-ui-fabric-react/lib/Separator';
+
+import { RouterPaths } from '../shared/constants';
+
+import { AppContext } from './VirtualAssistantCreationModal';
+import { DialogFooterWrapper } from './dialogFooterWrapper';
+
+// -------------------- Styles -------------------- //
+
+const categoryTextStyling = css`
+  display: block;
+  margin-top: 10px;
+`;
+
+const entryText = (bold: boolean) => css`
```

Comment:
Prefer suffix of Style (or Css) for any css variable and then using a name for any styled component.